_includes/head.html

```diff
@@ -29,7 +29,19 @@
 <script src="{{ "js/toc.js" }}"></script>
 <script src="{{ "js/customscripts.js" }}"></script>
 
-<link rel="shortcut icon" href="{{ "images/favicon.ico"  }}">
+<link rel="shortcut icon" href="{{ "https://assets.tidalcloud.com/favicons/tidal-favicon-32x32.png"  }}">
```

Comment:
We should be able to drop this
```suggestion
```

<img width="303" alt="image" src="https://user-images.githubusercontent.com/6868388/204605752-780e036b-0745-48cb-b686-cba9e52568c1.png">

From the `icon` seciont https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

Reply:
Oh nice! Thanks